complearn-genbank/Dockerfile

```diff
@@ -1,17 +1,27 @@
-FROM node:18-alpine
+# Build stage
+FROM node:18-alpine as builder
 WORKDIR /app
 
-# Copy package files
 COPY package*.json ./
 
-# Install dependencies
 RUN npm install
 
-# Copy source code
 COPY . .
 
```

Comment:
why do you remove this?
the dist has to be removed everytime we build docker image and rebuild again using npx tsc